<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research - Amy K. Webster</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><strong>Amy K. Webster</strong>   genetics & epigenetics of quantitative traits</a>
									<ul class="icons">
										<li><a href="https://twitter.com/itsamywebster" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
									<!--	<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon brands fa-snapchat-ghost"><span class="label">Snapchat</span></a></li>
										<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="#" class="icon brands fa-medium-m"><span class="label">Medium</span></a></li>-->
									</ul>
								</header>

							<!-- Content -->
								<section>
									<header class="main">
										<h1>Research</h1>
									</header>

								<!--	<span class="image main"><img src="images/pic11.jpg" alt="" /></span>-->

									<p>Why individuals are different from each other is one of life's fundamental puzzles. A given individual's phenotype may be driven by a combination of genetic and present environmental conditions, along with prior environmental experiences, development, or other unknown causes. A major goal of my research is to understand the relative influence of different sources of variation and the implications of these various sources throughout the lifespan of individuals and on the evolution of populations. I use both experimental and theoretical approaches to address these questions, and I primarily use the nematode <i>C. elegans </i>for the former.
 </p>
					<!--				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis dapibus rutrum facilisis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Etiam tristique libero eu nibh porttitor fermentum. Nullam venenatis erat id vehicula viverra. Nunc ultrices eros ut ultricies condimentum. Mauris risus lacus, blandit sit amet venenatis non, bibendum vitae dolor. Nunc lorem mauris, fringilla in aliquam at, euismod in lectus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. In non lorem sit amet elit placerat maximus. Pellentesque aliquam maximus risus, vel sed vehicula.</p>
									<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque venenatis dolor imperdiet dolor mattis sagittis. Praesent rutrum sem diam, vitae egestas enim auctor sit amet. Pellentesque leo mauris, consectetur id ipsum sit amet, fersapien risus, commodo eget turpis at, elementum convallis elit. Pellentesque enim turpis, hendrerit tristique lorem ipsum dolor.</p>
-->	<hr class="major" />
	<h2>Why <i>C. elegans</i>?</h2>
		<div class="row">
										<div class="col-6 col-12-small">
												<p>The nematode <i>C. elegans</i> is well-suited to support research in the area of individual variation. <i>C. elegans</i> primarily exist as hermaphrodites (with rare males) and do not exhibit inbreeding depression. Each individual hermaphrodite can produce roughly 300 progeny in a matter of days. In the case of inbred lines, each worm produces genetically identical progeny, making it relatively easy to generate thousands of genetically identical worms. These worms are microscopic and transparent, can be genetically altered via CRISPR or other genetic engineering approaches, and can be grown in liquid or on plates. The <i>C. elegans</i> genome was the first animal genome to be sequenced, and it is well annotated. In addition, the nematode community has collected and sequenced many genetically-distinct wild isolates within <i>C. elegans</i> as well as related nematodes in the <i>Caenorhabditis</i> genus. In sum, both the genetic and environmental conditions across generations can be carefully controlled with this system, and the worm community has developed resources to facilitate sophisticated analysis of the causes and consequences of variation.  </p>
											</div>
											<!--<div class="col-6 col-12-small">-->
													<div class="col-6"><span class="image fit"><img src="images/worms.png" alt="" /></span></div></div>
									
										<hr class="major" />
									<h2>Postdoctoral Resarch: Why are genetically identical individuals distinct?</h2>
									<p> To gain traction on the question of why individuals are different from each other, I have honed in on part of this question: how are individuals different from each other, even when they are genetically identical? Many identical human twins exhibit divergent disease outcomes or have different traits, even when raised in similar environments, for reasons that remain poorly understood. Do micro-environmental differences or other causes lead to these differences? Genetic background cannot be controlled in humans, and for many other species. Using <i>C. elegans</i>, I am focusing on quantitative reproductive traits that are relative to fitness, for which genetically identical individuals that develop in the same environment exhibit substantial variation. I hypothesized that gene-regulatory differences that may be driven by epigenetic modifications have substantial explanatory power for these differences. I am using single-individual transcriptomics and epigenomics to dissect these traits, and have identified several genes for which 'natural expression variation' underlies differences in reproductive traits. In addition, I am using theory to understand when gene-regulatory variation can be maintained in populations (fairly frequently for certain parameters), and when such variation may drive the maintenence of genetic variation (more rarely). These projects will lay the groundwork to understand how gene-regulatory variation may interact with other factors.</p>
									<hr class="major" />


									<h2>PhD Research: How do animals endure starvation? </h2>
									<p>For my dissertation research, I focused on understanding how <i>C. elegans</i> endure starvation. Survival in the face of limited food availability is a fitness challenge faced by virtually all organisms. <i>C. elegans</i> is well adapted to survive starvation conditions, as its development is coupled to food availability. When embryos hatch as L1 larvae in the absence of food, they remain developmentally arrested in the first larval stage, and this is termed L1 arrest. <i>C. elegans</i> larvae can survive for weeks without food during L1 arrest. Remarkably, if an arrested L1 encounters food, it recovers and the time it spent as an L1 does not count against its subsequent lifespan. It is thus considered an “ageless” state. However, we know that during L1 arrest larvae accumulate various markers of aging that are reversed upon recovery. In addition, extended time spent in L1 arrest leads to fitness consequences later in life and in subsequent generations. Further, some genes already implicated in regulating survival during L1 arrest are broadly conserved and play roles as tumor suppressors in human cancers. Given the significance of L1 arrest for understanding processes as broad as aging and intergenerational inheritance, in addition to its ecological and biomedical relevance, it is of great interest to understand how starvation resistance is regulated. To this end, I analyzed how starvation resistance is impacted by 1) transgenerational effects of ancestral starvation, 2) transcriptional regulation, and 3) genes that differ in natural populations. Each of these projects is elaborated on below. </p>
									

              	<div class="row">
											<div class="col-6 col-12-small">
												<h3>Transgenerational effects of starvation</h3>
												<p>I found worms that experience extended dauer diapause, an environmentally-induced developmental arrest, produce great-grand progeny that exhibit increased starvation resistance as larvae and live longer as adults compared to genetically identical worms with unstarved ancestors. These worms also exhibit gene expression differences consistent with a small, genome-wide effect on the nutrient response. This effect is in contrast to the first generation following extended dauer diapause, in which worms exhibit increased inter-individual variation and decreased starvation resistance. These results suggest that ancestral environment impacts phenotypes of fitness-relevant traits several generations later. Read more about this work in our <a href="https://academic.oup.com/genetics/article/210/1/263/6088081?login=false">Genetics</a> paper. A figure from this paper showing the phenotypic effects of ancestral starvation is shown to the right.</p>
											</div>
											<!--<div class="col-6 col-12-small">-->
													<div class="col-6"><span class="image fit"><img src="images/TransgenFig.jpg" alt="" /></span></div>
										<!--	</div>-->
	<div class="row">
											<div class="col-6 col-12-small">
												<h3>Gene-regulatory dynamics of starvation</h3>
												<p>Worms can survive L1 arrest for about two weeks, but temporal transcriptional dynamics beyond the first 24 hours and their importance for survival have not been previously characterized. I analyzed gene expression profiles of L1 larvae from immediately before hatching until death. Although worms starved different lengths of time are in the same developmental stage, mRNA transcriptional profiles continue to change even late in starvation, though the rate of gene expression change slows. Using tissue-specific and temporal depletion of RNA Pol II, we found gene expression differences occurring deep in starvation are due to a combination of reduced somatic transcription and long-term maintenance of germline transcript stability. While transcription is required in the soma early in starvation, it is (perhaps surprisingly) not required late in starvation. Germline transcriptional quiescence, already known to occur early in starvation, is maintained even deep into starvation, and this is required for reproduction upon recovery from extended starvation. You can read more about this work in our paper in <a href="https://www.cell.com/cell-reports/pdf/S2211-1247(22)01323-7.pdf">Cell Reports</a>. We were also featured on the cover, shown to the right, which shows individual gene expression trajectories for genes throughout starvation. The green and purple lines are clusters enriched for germline-expressed genes. </p>
											</div>
											<!--<div class="col-6 col-12-small">-->
													<div class="col-6"><span class="image fit"><img src="images/CellReportsCover.jpg" alt="" /></span></div>
										<!--	</div>-->
										
											<div class="row">
											<div class="col-6 col-12-small">
												<h3>Natural genetic variation underlying starvation resistance</h3>
												<p>Within the <i>C. elegans</i> species, many genetically distinct wild isolates have been collected and whole-genome sequenced. I utilized many wild strains from this resource that represent the diversity of the species and used two high-throughput sequencing approaches to phenotype these strains for starvation resistance simtultaneously. In collaboration with others, I identified strains that are particularly sensitive or resistant to starvation and uncovered regions of the genome that are important for affecting starvation resistance. The first iteration of this project using RAD-seq for phenotyping-by-sequencing has been published in <a href="https://academic.oup.com/g3journal/article/9/10/3477/6026714?login=false">G3</a>. For the next iteration, we used MIP-seq to sequence only at sites in the genome for which each strain had a unique single-nucleotide polymorphism (SNP), thus reducing the amount of sequencing needed. We ultimately identified multiple members of the insulin receptor-like domain (<i>irld</i>) family containing variation (shown in the figure to the right) that affects starvation resistance and act in the insulin-like signaling pathway. You can read more about this in our paper published in <a href="https://elifesciences.org/articles/80204">eLife</a>.</p>
											</div>
											<!--<div class="col-6 col-12-small">-->
													<div class="col-6"><span class="image fit"><img src="images/MIPseq_figure.png" alt="" /></span></div>
												
										<!--	</div>-->
										
										
								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Search -->
								<section id="search" class="alt">
									<form method="post" action="#">
										<input type="text" name="query" id="query" placeholder="Search" />
									</form>
								</section>

							<!-- Menu -->
									<nav id="menu">
									<header class="major">
										<h2>Menu</h2>
									</header>
									<ul>
										<li><a href="index.html">Homepage</a></li>
										<li><a href="research.html">Research</a></li>
										<!--<li><a href="elements.html">Elements</a></li> -->
											<li>
											<span class="opener">Publications</span>
											<ul>
												<li><a href="https://scholar.google.com/citations?hl=en&user=U1PssuYAAAAJ">Google Scholar</a></li>
												<li><a href="https://pubmed.ncbi.nlm.nih.gov/?term=webster+amy+k%5Bfull+author+name%5D&sort=date">PubMed</a></li>
											</ul>
										</li>
										<li><a href="CV_Webster.pdf">CV (pdf)</a></li>
										<li>
											<span class="opener">Data Exploration</span>
											<ul>
												<li><a href="https://awebster.shinyapps.io/shinyapp/">RShiny App: Gene expression throughout starvation</a></li>
												<li><a href="https://github.com/amykwebster">Github repositories</a></li>
												<li><a href="https://www.ncbi.nlm.nih.gov/gds/?term=webster+ak%5BAuthor%5D">NCBI Gene Expression Omnibus</a></li>
											</ul>
										</li>
								
										<!--		<li><a href="#">Adipiscing</a></li> -->
									
								<!--		<li><a href="#">Maximus Erat</a></li>
										<li><a href="#">Sapien Mauris</a></li>
										<li><a href="#">Amet Lacinia</a></li> -->
									</ul>
								</nav>

							<!-- Section -->
								<section>
									<header class="major">
									<!--	<h2>Ante interdum</h2>-->
									</header>
									<div class="mini-posts">
										<article>
									<!--		<a href="#" class="image"><img src="images/pic07.jpg" alt="" /></a>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore aliquam.</p>
										</article>
										<article>
											<a href="#" class="image"><img src="images/pic08.jpg" alt="" /></a>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore aliquam.</p>
										</article>
										<article>
											<a href="#" class="image"><img src="images/pic09.jpg" alt="" /></a>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore aliquam.</p>
										</article>
									</div>
									<ul class="actions">
										<li><a href="#" class="button">More</a></li>
									</ul>
								</section>-->

							<!-- Section -->
								<section>
									<header class="major">
											<h2>Get in touch</h2>
									</header>
									<p>If you're interested in my work or would like to collaborate, please reach out!</p>
									<ul class="contact">
										<li class="icon solid fa-envelope"><a href="#">awebst10@uoregon.edu</a></li>
									<!--	<li class="icon solid fa-phone">(000) 000-0000</li> -->
										<li class="icon solid fa-home">Institute of Ecology and Evolution<br />
										University of Oregon<br />
										Eugene, OR 97401</li>
									</ul>
								</section>

							<!-- Footer -->
								<footer id="footer">
									<p class="copyright">&copy; Amy Webster. All rights reserved. 
									<!--Demo Images: <a href="https://unsplash.com">Unsplash</a>. -->
									Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
								</footer>

						</div>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>